draft-ietf-tls-tls13.md

```diff
@@ -1884,7 +1885,7 @@ Cookies serve two primary purposes:
 
 - Allowing the server to offload state to the client, thus allowing it to send
   a HelloRetryRequest without storing any state. The server does this by
-  pickling that post-ClientHello hash state into the cookie (protected
+  setting the post-ClientHello hash state as the cookie (protected
```

Comment:
I don't think this is quite right, as the server can (and should) put other state into the cooke as well as just the hash state.  "pickling" is perhaps a needless term of art, though, and "including [state] in the cookie" would do just fine.